2020-36-03 15:36:09-__init__-DEBUG-Using selector: SelectSelector
2020-42-03 15:42:25-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-42-03 15:42:25-__init__-DEBUG-Using selector: SelectSelector
2020-42-03 15:42:32-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-42-03 15:42:32-__init__-DEBUG-Using selector: SelectSelector
2020-42-03 15:42:35-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-42-03 15:42:35-__init__-DEBUG-Using selector: SelectSelector
2020-42-03 15:42:40-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-42-03 15:42:40-__init__-DEBUG-Using selector: SelectSelector
2020-42-03 15:42:43-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-42-03 15:42:43-__init__-DEBUG-Using selector: SelectSelector
2020-42-03 15:42:52-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-42-03 15:42:52-__init__-DEBUG-Using selector: SelectSelector
2020-53-03 15:53:12-__init__-DEBUG-Using selector: SelectSelector
2020-54-03 15:54:39-__init__-DEBUG-Using selector: SelectSelector
2020-57-03 15:57:45-log_response-WARNING-Not Found: /apis/filter_messages/
2020-02-03 16:02:33-log_response-ERROR-Internal Server Error: /apis/encrypt_message/
Traceback (most recent call last):
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "D:\Study\School\e2e\app\djangoapp\backend\views.py", line 337, in encrypt_message
    "ciphertext": ct.encode("utf-8"),
AttributeError: 'bytes' object has no attribute 'encode'
2020-02-03 16:02:51-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-02-03 16:02:51-__init__-DEBUG-Using selector: SelectSelector
2020-02-03 16:02:54-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-02-03 16:02:54-__init__-DEBUG-Using selector: SelectSelector
2020-02-03 16:02:58-log_response-ERROR-Internal Server Error: /apis/encrypt_message/
Traceback (most recent call last):
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "D:\Study\School\e2e\app\djangoapp\backend\views.py", line 337, in encrypt_message
    "ciphertext": ct.decode("utf-8"),
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xd9 in position 1: invalid continuation byte
2020-03-03 16:03:11-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-03-03 16:03:11-__init__-DEBUG-Using selector: SelectSelector
2020-03-03 16:03:23-log_response-ERROR-Internal Server Error: /apis/encrypt_message/
Traceback (most recent call last):
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "D:\Study\School\e2e\app\djangoapp\backend\views.py", line 342, in encrypt_message
    return JsonResponse(result)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\http\response.py", line 587, in __init__
    data = json.dumps(data, cls=encoder, **json_dumps_params)
  File "C:\Users\Pisho\AppData\Local\Programs\Python\Python37\lib\json\__init__.py", line 238, in dumps
    **kw).encode(obj)
  File "C:\Users\Pisho\AppData\Local\Programs\Python\Python37\lib\json\encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "C:\Users\Pisho\AppData\Local\Programs\Python\Python37\lib\json\encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\serializers\json.py", line 104, in default
    return super().default(o)
  File "C:\Users\Pisho\AppData\Local\Programs\Python\Python37\lib\json\encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type bytes is not JSON serializable
2020-03-03 16:03:54-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-03-03 16:03:54-__init__-DEBUG-Using selector: SelectSelector
2020-04-03 16:04:02-log_response-ERROR-Internal Server Error: /apis/encrypt_message/
Traceback (most recent call last):
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "D:\Study\School\e2e\app\djangoapp\backend\views.py", line 342, in encrypt_message
    return JsonResponse(result)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\http\response.py", line 587, in __init__
    data = json.dumps(data, cls=encoder, **json_dumps_params)
  File "C:\Users\Pisho\AppData\Local\Programs\Python\Python37\lib\json\__init__.py", line 238, in dumps
    **kw).encode(obj)
  File "C:\Users\Pisho\AppData\Local\Programs\Python\Python37\lib\json\encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "C:\Users\Pisho\AppData\Local\Programs\Python\Python37\lib\json\encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\serializers\json.py", line 104, in default
    return super().default(o)
  File "C:\Users\Pisho\AppData\Local\Programs\Python\Python37\lib\json\encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type bytes is not JSON serializable
2020-04-03 16:04:13-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-04-03 16:04:13-__init__-DEBUG-Using selector: SelectSelector
2020-04-03 16:04:16-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-04-03 16:04:16-__init__-DEBUG-Using selector: SelectSelector
2020-04-03 16:04:55-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-04-03 16:04:55-__init__-DEBUG-Using selector: SelectSelector
2020-05-03 16:05:00-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-05-03 16:05:00-__init__-DEBUG-Using selector: SelectSelector
2020-05-03 16:05:08-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-05-03 16:05:08-__init__-DEBUG-Using selector: SelectSelector
2020-05-03 16:05:17-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-05-03 16:05:17-__init__-DEBUG-Using selector: SelectSelector
2020-05-03 16:05:22-log_response-ERROR-Internal Server Error: /apis/encrypt_message/
Traceback (most recent call last):
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "D:\Study\School\e2e\app\djangoapp\backend\views.py", line 330, in encrypt_message
    "kdf_next": binascii.unhexlify(kdf_out[:32]),
binascii.Error: Non-hexadecimal digit found
2020-05-03 16:05:35-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-05-03 16:05:35-__init__-DEBUG-Using selector: SelectSelector
2020-06-03 16:06:06-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-06-03 16:06:06-__init__-DEBUG-Using selector: SelectSelector
2020-06-03 16:06:09-log_response-ERROR-Internal Server Error: /apis/encrypt_message/
Traceback (most recent call last):
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "D:\Study\School\e2e\app\djangoapp\backend\views.py", line 339, in encrypt_message
    return JsonResponse(result)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\http\response.py", line 587, in __init__
    data = json.dumps(data, cls=encoder, **json_dumps_params)
  File "C:\Users\Pisho\AppData\Local\Programs\Python\Python37\lib\json\__init__.py", line 238, in dumps
    **kw).encode(obj)
  File "C:\Users\Pisho\AppData\Local\Programs\Python\Python37\lib\json\encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "C:\Users\Pisho\AppData\Local\Programs\Python\Python37\lib\json\encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\serializers\json.py", line 104, in default
    return super().default(o)
  File "C:\Users\Pisho\AppData\Local\Programs\Python\Python37\lib\json\encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type bytes is not JSON serializable
2020-06-03 16:06:30-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-06-03 16:06:30-__init__-DEBUG-Using selector: SelectSelector
2020-06-03 16:06:46-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-06-03 16:06:46-__init__-DEBUG-Using selector: SelectSelector
2020-06-03 16:06:54-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-06-03 16:06:54-__init__-DEBUG-Using selector: SelectSelector
2020-07-03 16:07:02-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-07-03 16:07:02-__init__-DEBUG-Using selector: SelectSelector
2020-07-03 16:07:10-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-07-03 16:07:10-__init__-DEBUG-Using selector: SelectSelector
2020-07-03 16:07:18-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-07-03 16:07:18-__init__-DEBUG-Using selector: SelectSelector
2020-08-03 16:08:07-log_response-ERROR-Internal Server Error: /apis/encrypt_message/
Traceback (most recent call last):
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "D:\Study\School\e2e\app\djangoapp\backend\views.py", line 339, in encrypt_message
    return JsonResponse(result)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\http\response.py", line 587, in __init__
    data = json.dumps(data, cls=encoder, **json_dumps_params)
  File "C:\Users\Pisho\AppData\Local\Programs\Python\Python37\lib\json\__init__.py", line 238, in dumps
    **kw).encode(obj)
  File "C:\Users\Pisho\AppData\Local\Programs\Python\Python37\lib\json\encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "C:\Users\Pisho\AppData\Local\Programs\Python\Python37\lib\json\encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\serializers\json.py", line 104, in default
    return super().default(o)
  File "C:\Users\Pisho\AppData\Local\Programs\Python\Python37\lib\json\encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type type is not JSON serializable
2020-08-03 16:08:34-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-08-03 16:08:34-__init__-DEBUG-Using selector: SelectSelector
2020-08-03 16:08:36-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-08-03 16:08:36-__init__-DEBUG-Using selector: SelectSelector
2020-08-03 16:08:53-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-08-03 16:08:53-__init__-DEBUG-Using selector: SelectSelector
2020-08-03 16:08:57-log_response-ERROR-Internal Server Error: /apis/encrypt_message/
Traceback (most recent call last):
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "D:\Study\School\e2e\app\djangoapp\backend\views.py", line 340, in encrypt_message
    return JsonResponse(result)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\http\response.py", line 587, in __init__
    data = json.dumps(data, cls=encoder, **json_dumps_params)
  File "C:\Users\Pisho\AppData\Local\Programs\Python\Python37\lib\json\__init__.py", line 238, in dumps
    **kw).encode(obj)
  File "C:\Users\Pisho\AppData\Local\Programs\Python\Python37\lib\json\encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "C:\Users\Pisho\AppData\Local\Programs\Python\Python37\lib\json\encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\serializers\json.py", line 104, in default
    return super().default(o)
  File "C:\Users\Pisho\AppData\Local\Programs\Python\Python37\lib\json\encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type bytes is not JSON serializable
2020-09-03 16:09:05-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-09-03 16:09:05-__init__-DEBUG-Using selector: SelectSelector
2020-09-03 16:09:30-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-09-03 16:09:30-__init__-DEBUG-Using selector: SelectSelector
2020-09-03 16:09:35-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-09-03 16:09:35-__init__-DEBUG-Using selector: SelectSelector
2020-13-03 16:13:57-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-13-03 16:13:57-__init__-DEBUG-Using selector: SelectSelector
2020-14-03 16:14:00-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-14-03 16:14:00-__init__-DEBUG-Using selector: SelectSelector
2020-14-03 16:14:16-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-14-03 16:14:16-__init__-DEBUG-Using selector: SelectSelector
2020-14-03 16:14:19-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-14-03 16:14:19-__init__-DEBUG-Using selector: SelectSelector
2020-15-03 16:15:15-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-15-03 16:15:15-__init__-DEBUG-Using selector: SelectSelector
2020-15-03 16:15:22-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-15-03 16:15:22-__init__-DEBUG-Using selector: SelectSelector
2020-15-03 16:15:35-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-15-03 16:15:35-__init__-DEBUG-Using selector: SelectSelector
2020-15-03 16:15:43-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-15-03 16:15:43-__init__-DEBUG-Using selector: SelectSelector
2020-17-03 16:17:15-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-17-03 16:17:15-__init__-DEBUG-Using selector: SelectSelector
2020-17-03 16:17:48-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-17-03 16:17:48-__init__-DEBUG-Using selector: SelectSelector
2020-17-03 16:17:50-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-17-03 16:17:50-__init__-DEBUG-Using selector: SelectSelector
2020-18-03 16:18:09-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-18-03 16:18:09-__init__-DEBUG-Using selector: SelectSelector
2020-18-03 16:18:11-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-18-03 16:18:11-__init__-DEBUG-Using selector: SelectSelector
2020-18-03 16:18:37-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-18-03 16:18:37-__init__-DEBUG-Using selector: SelectSelector
2020-18-03 16:18:41-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-18-03 16:18:41-__init__-DEBUG-Using selector: SelectSelector
2020-19-03 16:19:28-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-19-03 16:19:28-__init__-DEBUG-Using selector: SelectSelector
2020-20-03 16:20:03-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-20-03 16:20:03-__init__-DEBUG-Using selector: SelectSelector
2020-20-03 16:20:06-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-20-03 16:20:06-__init__-DEBUG-Using selector: SelectSelector
2020-20-03 16:20:08-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-20-03 16:20:08-__init__-DEBUG-Using selector: SelectSelector
2020-20-03 16:20:42-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-20-03 16:20:42-__init__-DEBUG-Using selector: SelectSelector
2020-20-03 16:20:44-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-20-03 16:20:44-__init__-DEBUG-Using selector: SelectSelector
2020-20-03 16:20:49-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-20-03 16:20:49-__init__-DEBUG-Using selector: SelectSelector
2020-20-03 16:20:51-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-20-03 16:20:51-__init__-DEBUG-Using selector: SelectSelector
2020-21-03 16:21:21-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-21-03 16:21:21-__init__-DEBUG-Using selector: SelectSelector
2020-21-03 16:21:33-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-21-03 16:21:33-__init__-DEBUG-Using selector: SelectSelector
2020-21-03 16:21:51-__init__-DEBUG-Using selector: SelectSelector
2020-46-03 16:46:11-__init__-DEBUG-Using selector: SelectSelector
2020-46-03 16:46:15-__init__-DEBUG-Using selector: SelectSelector
2020-49-03 16:49:11-__init__-DEBUG-Using selector: SelectSelector
2020-57-03 16:57:40-__init__-DEBUG-Using selector: SelectSelector
2020-26-03 17:26:40-__init__-DEBUG-Using selector: SelectSelector
2020-36-03 17:36:35-log_response-WARNING-Not Found: /favicon.ico
2020-43-03 17:43:38-__init__-DEBUG-Using selector: SelectSelector
2020-44-03 17:44:07-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-44-03 17:44:07-__init__-DEBUG-Using selector: SelectSelector
2020-44-03 17:44:15-log_response-ERROR-Internal Server Error: /apis/store_message/
Traceback (most recent call last):
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "D:\Study\School\e2e\app\djangoapp\backend\views.py", line 55, in store_message
    messages.objects.creat(
AttributeError: 'Manager' object has no attribute 'creat'
2020-44-03 17:44:20-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-44-03 17:44:20-__init__-DEBUG-Using selector: SelectSelector
2020-44-03 17:44:35-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-44-03 17:44:35-__init__-DEBUG-Using selector: SelectSelector
2020-44-03 17:44:37-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-44-03 17:44:37-__init__-DEBUG-Using selector: SelectSelector
2020-44-03 17:44:52-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-44-03 17:44:52-__init__-DEBUG-Using selector: SelectSelector
2020-44-03 17:44:55-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-44-03 17:44:55-__init__-DEBUG-Using selector: SelectSelector
2020-45-03 17:45:00-log_response-ERROR-Internal Server Error: /apis/store_message/
Traceback (most recent call last):
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\core\handlers\base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Pisho\AppData\Roaming\Python\Python37\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "D:\Study\School\e2e\app\djangoapp\backend\views.py", line 56, in store_message
    fromUserid=post_data["fromUserid"], toUserid=post_data["toUserid"], kdf_next=post_data["kdf_next"], EphemeralPub=post_data["EphemeralPub"], plaintext=post_data["plaintext"])
KeyError: 'fromUserid'
2020-45-03 17:45:41-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-45-03 17:45:41-__init__-DEBUG-Using selector: SelectSelector
2020-45-03 17:45:45-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-45-03 17:45:45-__init__-DEBUG-Using selector: SelectSelector
2020-46-03 17:46:09-trigger_reload-INFO-D:\Study\School\e2e\app\djangoapp\backend\views.py changed, reloading.
2020-46-03 17:46:09-__init__-DEBUG-Using selector: SelectSelector
